finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,14 @@
+
+for (var i = 1; i <= 100; i++) {
+    if(i % 3 === 0 && i % 5 === 0 && i > 0) {
```

Comment:
@Jacksonmcguire - The last part of this `if` expression is unnecessary - `&& i > 0` - because it will always be true, since you declared `i = 1` in the first part of the for loop, and it's incremented on each iteration.